code/dqe/constructcheck.q

```diff
@@ -3,7 +3,6 @@ constructcheck:{[construct;chktype]
   chkfunct:{system x," ",string $[null y;`;y]};
   dict:`table`variable`view`function!chkfunct@/:"avbf";
   .lg.o[`dqe;"checking if ", (s:string construct)," ",(s2:string chktype), " exists"];
-  $[construct in dict[chktype][];
-    (1b;s," ",s2," exists");
-    (0b;s," ",s2," missing from process")]
+  c:construct in dict[chktype][];
+  (c;s," ",s2," ",$[c:construct in dict[chktype][];"exists";"missing from process"];$[chktype=`table;construct in tables[];enlist value construct])
```

Comment:
Following code duplicated on 6 + 7, just need it in one place: `c:construct in dict[chktype][]`